CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Detect original video framerate and use it in lambda encode
+- Limit video encoding resolution to the original one
```

Comment:
why not get the available resolutions from the urls field?

Reply:
It avoids some computation, I think it's easier like this.

Reply:
I removed the `resolutions` property in the seralizers and everything is computed now from urls.